pkg/slayers/path/scion/raw_spec.gobra

```diff
@@ -402,13 +392,19 @@ pure func (s *Raw) absPkt(raw []byte) (res io.IO_pkt2) {
 		}
 }
 
+ghost
+decreases
+pure func NotFullyTraversedPkt(pkt io.IO_pkt2) bool {
```

Comment:
I would make this a pure method instead of a function (might require moving this to the package where IO_pkt2 is defined), and I would rename this to `PathNotFullyTraversed`

Reply:
Done.